### PR TITLE
django lab02

### PR DESCRIPTION
**Pull request by webavant**

### CODE REVIEW

File: code/tony/django/lab02/url_shortener/views.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    ct += 1
+    return get_base(int(x / n) + int(x % n), ct=ct)
+
+
+def get_str(the_id, min_length=5):
+    """Get a base64 string that tries to match a complexity 
+    based upon the amount of database entries."""
+
+    b64 = list('0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ-_')
+    "There's probably a more pythonic way to get this list, but I don't know how to do it."
```
Comment (iammattwalsh) [inline code/tony/django/lab02/url_shortener/views.py:58]: There's nothing wrong with this, but you could have imported strings from the `string` built-in module.

`from string import ascii_letters, digits`
`...`
`    b64  = list(ascii_letters + digits + '-_')`

`string` also has `punctuation` if you wanted to include more than just - and _.

**Review by iammattwalsh — APPROVED** [review]
Really nice work, and good use of class-based views!

File: code/tony/django/lab02/url_shortener/views.py (hunk 1 above)
Comment (webavant) [reply code/tony/django/lab02/url_shortener/views.py:58]: Yassss

**Comment by webavant** [comment]
> Really nice work, and good use of class-based views!

Thanks! I know there's some more sloppy stuff in here. I'm looking forward to having more time to  better familiarize myself with standards and best practices.

**Pull request merged**